lastuser_ui/forms/profile.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 
+from flask_wtf import RecaptchaField
```

Comment:
Add this field to Baseframe. We shouldn't be importing anything from `flask_wtf`